app/site/controllers.py

```diff
@@ -52,6 +51,7 @@ def logout():
       302:
         description: Load the Home Page
     """
+    g.current_user = None
     resp = make_response(render_template("logout.html", title='Logout'), 200)
```

Comment:
It would make more sense to set a flash message and redirect them to the home page.